tests/app/questionnaire_state/test_relationship_state_question.py

```diff
@@ -130,9 +149,64 @@ def test_should_set_other_person_on_widget(self):
         relationship_state_question.answers = [answer_state]
 
         with patch('app.questionnaire_state.relationship_state_question.get_answer_store') as get_answer_store:
-            get_answer_store().filter = Mock(return_value=self.household_answers)
+            get_answer_store().filter = Mock()
+            get_answer_store().filter.side_effect = [self.household_answers, self.last_names]
             # When
             relationship_state_question.update_state({})
 
         # Then
         self.assertEqual(relationship_state_question.children[0].schema_item.widget.other_person, 'Joe Bloggs')
+
+    def test_should_display_single_name_if_no_surname_provided(self):
```

Comment:
Should we split this out into two tests for clarity I think
`test_should_display_first_name_if_no_surname_provided`
`test_should_display_surname_if_no_first_name_provided`